src/identity.ts

```diff
@@ -151,6 +150,33 @@ export const getIdentities = async (
   };
 };
 
+/**
+ * Deletes all matching identities.
+ *
+ * @param serviceId The service to search for matching identities within.
+ * @param idsOrTags Either a { userId, tenantId } that can be used to search for a matching identity, or a set of
+ * tags that will be used to search. Pass 'null' to delete all identities for a service.
+ */
+export const deleteIdentities = async (
```

Comment:
Change the manual tests to use this, during their beforeAll cleanup, so that it gets exercised.

Reply:
fixed